tasks/utils/getConfig.js

```diff
@@ -0,0 +1,28 @@
+const { tryRequireSync } = require('@keg-hub/jsutils/src/node')
+const { deepMerge } = require('@keg-hub/jsutils')
+const os = require('os')
+const path = require('path')
+const appRoot = require('app-root-path').path
+
+const HOME_DIR = os.homedir()
+
+// paths to tap-watchtower configuration files
+const configPaths = [
+  path.join(appRoot,'configs/watchtower.config.json'),
+  path.join(appRoot,'configs/watchtower.config.js'),
+  path.join(HOME_DIR, 'watchtower.config.json'),
+  path.join(HOME_DIR,'watchtower.config.js'),
```

Comment:
We should also add a check inside of the .kegConfig folder

Reply:
updated 👍